app/models/root-store/setup-root-store.ts

```diff
@@ -53,6 +53,18 @@ function createRootStore(env: Environment, data: any = {}) {
   env.authCoreAPI.callbacks.unauthorized = rootStore.handleUnauthenticatedError
   env.likeCoAPI.config.onUnauthenticated = rootStore.handleUnauthenticatedError
   env.likerLandAPI.config.onUnauthenticated = rootStore.handleUnauthenticatedError
+
+  // Reset app rating cooldown at first open
+  if (!rootStore.userStore.appRatingCooldown) {
+    rootStore.userStore.resetAppRatingCooldown()
+  } else if (
+    rootStore.userStore.ratedAppVersion &&
+    rootStore.userStore.ratedAppVersion !== env.appConfig.getValue("APP_VERSION")
```

Comment:
Maybe we should judge major version changes (and maybe even more than a few major version changes) for less disturbing UX